### ISSUE TITLE
Localization (0.0.5)

### ISSUE THREAD

**Issue by hensm** [hensm/fx_cast#99]
@rimrul, @RAVMN, @Vistaus 

Updated list if you want to provide new strings:
https://github.com/hensm/fx_cast/blob/master/CONTRIBUTING.md#localizations

**Comment by ThaDaVos** [comment]
Where can we supply the translations?

**Comment by ThaDaVos** [comment]
Or even find them? :P

**Comment by hensm** [comment]
Locales are found at `ext/src/_locales/`:
https://github.com/hensm/fx_cast/tree/master/ext/src/_locales

More info:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales

**Comment by ThaDaVos** [comment]
According to this site: http://www.jsondiff.com/ there are 85 properties missing between both files, should I go over them all? Or completely redo the NL one based on the EN one?

**Comment by hensm** [comment]
It's okay, I already diffed them. They're listed in CONTRIBUTING at the link in the top comment.

**Comment by ThaDaVos** [comment]
Here you go :D
https://github.com/hensm/fx_cast/pull/100

**Comment by ThaDaVos** [comment]
Can't you run the left over Locales through Google Translate and Bing Translate, compare the translations and which match use them?

**Comment by hensm** [comment]
I don't know how reliable machine translation would be. It may not translate idiomatically (which I can't check) and some of the messages have very little contextual info to assist. I could probably do Swedish, but anything else is lost on me.

I've used Microsoft's [Terminology Collection](https://www.microsoft.com/en-us/language) in the past for simple things, but it's quite limited.

Side note, I found this translation tool that looks like it could make things much simpler:
https://lusito.github.io/web-ext-translator/?gh=https://github.com/hensm/fx_cast/

**Comment by ThaDaVos** [comment]
Like I thought, there are tools for it, I've seen them in the past, even successfully used - you could always place a English Disclaimer at the bottom telling people that it's machinely translated and if there are mistakes they can make an issue about it, or supply better translations through a PR - this way, you won't be hold up on translations.

**Comment by hensm** [comment]
It might be worth exploring. Still not sure I'd want to mix human/machine translations in a single localization. I'm more comfortable just releasing with partial localizations.

Here's an example for Dutch:
https://gist.github.com/hensm/a0c482720c20ecc36d343227bc718e6d

**Comment by ThaDaVos** [comment]
You can see it's translated by a machine sadly - it's to translated to direct

**Comment by hensm** [comment]
I suspected it wouldn't be great when I saw how different the results were compared to your translation.

I can schedule localizations better for future releases so I'm not waiting on anything. Also, the extension should be able to auto-update from the next release, so it's not a disaster if things aren't fully localized. Might have to bump to 1.0.0 for the versioning to indicate bridge compat for minor releases.
